### ISSUE TITLE
Panels display in the wrong order in mobile view

### ISSUE THREAD

**Issue by pmackay** [folklabs/openlocal#17]
Copied from https://github.com/folklabs/vassallandcoldharbour/issues/2.

This is an issue that's also a problem on the mobile view of HH, let alone OL and Vassall. Therefore it would probably repay some time fixing it or addressing it.

Basically, the panels are displaying in the wrong order on the mobile view of the site. Here, we can see the top of the home page on Vassall, which is displaying the Upcoming Events panel at the top of the page. This is wrong – the first panel displayed should be the main central column content, and Events should appear somewhat lower down the page.

I can go and write a list of which order the panels should appear in but, before I do, is it safe to assume that this is actually even possible? If not, how else can we do this? Could we adjust the display of panels based on media query or some other method?

**Comment by pmackay** [comment]
This https://www.drupal.org/project/breakpoint_panels would be worth looking at, but also should see if just hiding a colum/region would be enough.

**Comment by djwesto** [comment]
I don't think this is necessarily just going to be solved by hiding panels for mobile. 

My preferred approach would be to develop new Panopoly page templates with Foundation using their source ordering and push/pull classes to ensure each view: small / medium / large has the right order of panels.
